src/WebJobs.Extensions/Extensions/Core/ExecutionContext.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Azure.WebJobs
     public class ExecutionContext
     {
         /// <summary>
-        /// The job function invocation ID
+        /// Gets or sets the job function invocation ID
```

Comment:
We'll probably want to move this binding into core SDK for v2 - this is functionality that should be in core.